core/src/main/java/brooklyn/event/feed/AttributePollHandler.java

```diff
@@ -52,6 +54,7 @@
     @SuppressWarnings("rawtypes")
     private final AttributeSensor sensor;
     private final AbstractFeed feed;
+    private final boolean suppressDuplicates;
```

Comment:
A couple of reasons: first (lame) it's preserving backwards compatibility; second for performance-related metrics then values such as "total requests processed" matter - there are enrichers that calculate the diff between recent values to determine the average requests per second.

I suggest you take it to the mailing list for changing the default behaviour to always-suppress before we change that.